app/models/elastic_search_recommended_link.rb

```diff
@@ -23,29 +23,23 @@ def body
 
   def details
     {
-      title: @recommended_link.title,
-      link: @recommended_link.link,
-      description: @recommended_link.description,
+      title: title,
+      link: link,
+      description: description,
       format: format,
-      indexable_content: @recommended_link.keywords
+      indexable_content: indexable_content
     }
   end
 
   def id
     link
   end
 
-  def format
-    if URI(@recommended_link.link).host.downcase == "www.gov.uk"
-      'inside-government-link'
-    else
-      'recommended-link'
-    end
+  def indexable_content
+    @recommended_link.keywords
```

Comment:
I think we could also use the getter here instead of the instance variable.

This is looking pretty good, happy to merge after this 👍